source/partials/header/_desktop_navigation.html.erb

```diff
@@ -1,259 +1,39 @@
-            compact: true,
-            icon: 'megaphone-simple',
-            title: 'Media Center',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'smiley',
-            icon_hover: 'smiley-wink-fill',
-            title: 'Reviews',
-            href: '/TODO',
-          } %>
-        </div>
-
-        <div class="menu__column">
-          <div class="menu__title" role="separator">Work with Us</div>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'identification-card',
-            title: 'Partner Directory',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'handshake',
-            title: 'Become a Partner',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'share-network',
-            icon_hover: 'fire-fill',
-            title: 'Become an Affiliate',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'envelope-simple',
-            icon_hover: 'envelope-simple-open',
-            title: 'sales@sharesight.com',
-            href: 'mailto:sales@sharesight.com',
-          } %>
-        </div>
-
-        <div class="menu__column">
-          <div class="menu__title" role="separator">Resources</div>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'info',
-            title: 'Help Centre',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'code',
-            title: 'Sharesight API',
-            href: '/TODO',
-          } %>
-
-          <%= partial "partials/header/link", locals: {
-            compact: true,
-            icon: 'monitor-play',
-            icon_hover: 'monitor-fill',
-            title: 'Events & Webinars',
-            href: '/TODO',
-          } %>
-        </div>
-      </div>
-
-      <div class="menu__background menu__blog">
-        <a class="menu__label menu__title" role="separator" href="/blog">
-          <i class="ph-pencil-circle menu__icon" role="img" aria-label="From the Blog"></i>
-          <span class="menu__label-text">From the Blog</span>
-        </a>
-
-        <% data.blog.posts.values.first(3).each do |blog_post| %>
-          <%= partial "partials/header/link", locals: {
-            title: blog_post[:title],
-            href: post_url(blog_post),
-            truncate: 75,
-            compact: true
-          } %>
+<ul class="nav__menubar" role="menubar" aria-label="Sharesight Navigation Menu">
+  <% menus.each_with_index do |menu, index| %>
```

Comment:
We could always move this to a helper as well

Reply:
Not sure what you mean here…I think in the `-mobile` version, I split this into `_row` and `_column`.  Let me merge these together and re-request a fuller review (minus full tests) that I'll put into _testing_.